database/config.py

```diff
@@ -0,0 +1,26 @@
+### config.py
+
+# Scheme: "postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>" pragma: allowlist secret
+
+username = "postgres"  # pragma: allowlist secret
+
+password = ""  # pragma: allowlist secret
+
+IPadd = "localhost"  # pragma: allowlist secret
+
+portNum = "5432"  # pragma: allowlist secret
+
+RDBname = "thor_test_tmp"  # pragma: allowlist secret
+
+RELEASE_DATABASE_URL = (

```

Comment:
I liked how the components of the URL are broken down into smaller pieces. It makes the code easier to read.